### PR TITLE
Add contribution guidelines

### PR DESCRIPTION
**Pull request by ramya-subramanyam**
By creating this pull request you agree to the terms in CONTRIBUTING.md.
https://github.com/Infineon/.github/blob/master/CONTRIBUTING.md
--- DO NOT DELETE ANYTHING ABOVE THIS LINE ---

CONTRIBUTING.md also tells you what to expect in the PR process.

Description
Provide the information we need to review your PR. What problem does the pull request solve? "Bug fix" is not a good description.

Related Issue
If you opened an issue before creating the PR, point to it here.

Context
What do we need to know about your development environment, tools, target, and so on. Screenshots are always helpful if there is a UI element to this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,11 +28,31 @@ For installation & getting started instruction please refer to [our docs](https:
...
 
-Please visit our [PSoC products page](https://www.infineon.com/cms/de/product/microcontroller/32-bit-psoc-arm-cortex-microcontroller/) for datasheets, pin out diagrams, and additional technical information.
+Please visit our [PSOC products page](https://www.infineon.com/cms/de/product/microcontroller/32-bit-psoc-arm-cortex-microcontroller/) for datasheets, pin out diagrams, and additional technical information.
 
 ## Contributing
 
-We welcome community contributions! Please refer to our [contribution guidelines](CONTRIBUTING.md). Your help is greatly appreciated 👐
+We welcome community contributions! Contributions are what make the open source community such an amazing place to learn, inspire, and create. Any contributions you make are **greatly appreciated** 👐.
+
+If you have a suggestion that would make this better, please fork the repo and create a pull request. You can also simply open an issue with the tag "enhancement".
```
Comment (jaenrig-ifx) [inline README.md:37]: Very nice text :) 
Shouldn´t we ask contributors to ask the feature with code owners if the change is a "big"?

**Review by ederjc — APPROVED** [review]
Minor topics in the comments, great contribution, thanks!

**Review by IFX-Anusha — APPROVED** [review]

**Review by ederjc — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 [![code format check](https://github.com/Infineon/arduino-core-psoc/actions/workflows/code_formatting.yml/badge.svg)](https://github.com/Infineon/arduino-core-psoc/actions)
 [![code spell check](https://github.com/Infineon/arduino-core-psoc/actions/workflows/codespell.yml/badge.svg)](https://github.com/Infineon/arduino-core-psoc/actions)
 [![commit message check](https://github.com/Infineon/arduino-core-psoc/actions/workflows/commit_formatting.yml/badge.svg)](https://github.com/Infineon/arduino-core-psoc/actions)
 [![docs build](https://readthedocs.org/projects/arduino-core-psoc/badge/?version=latest)](https://arduino-core-psoc.readthedocs.io/en/latest/?badge=latest)
 
-This repository integrates Infineon's [PSoC microcontrollers](https://www.infineon.com/cms/de/product/microcontroller/32-bit-psoc-arm-cortex-microcontroller/) into the [Arduino ecosystem](https://www.arduino.cc/en/main/software).
+This repository integrates Infineon's [PSOC microcontrollers](https://www.infineon.com/cms/de/product/microcontroller/32-bit-psoc-arm-cortex-microcontroller/) into the [Arduino ecosystem](https://www.arduino.cc/en/main/software).
```
Comment (jaenrig-ifx) [inline README.md:9]: Missing the TM?

File: README.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 
 > [!WARNING]  
-> This project is work in progress. Currently not all PSoC products & features are supported in Arduino. Please refer to the docs for further information.
+> This project is work in progress. Currently not all PSOC products & features are supported in Arduino. Please refer to the docs for further information.
```
Comment (jaenrig-ifx) [inline README.md:12]: Missing the TM?

**Review by jaenrig-ifx — APPROVED** [review]

**Pull request merged**